multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/SystemParameters.java

```diff
@@ -133,6 +137,9 @@ private void putRoutingParameters(Module module, Map<String, Object> moduleParam
         }
         moduleSystemParameters.put(SupportedParameters.DEFAULT_URI, appendRoutePathIfPresent(defaultUri, moduleParameters));
 
+        String liveHostUri = getDefaultHost(module.getName()) + DEFAULT_LIVE_HOST_BASED_URI;
```

Comment:
Why don't reuse DEFAULT_HOST_BASED_URI constant to set DEFAULT_LIVE_URI parameter